.github/workflows/trigger-circle-ci-workflow.yml

```diff
@@ -9,8 +9,6 @@ on:
     branches:
       - next
       - main
-      - version-prerelease-from-**
-      - version-patch-from-**
```

Comment:
we don't need this as it should already be covered by "pull_request_target" > "synchronize"